### ISSUE TITLE
[Question] Open a specific teamspeak window

### ISSUE THREAD

**Issue by Bluscream** [Luch00/LxBTSC#34]
Is there a way to open the complaint list via the plugin API for example?

**Comment by Luch00** [comment]
Nope, as far as I'm aware the plugin api contains no functions that allow adding, removing, modifying or triggering any UI elements except for the plugin main menu and context menu items.

**Comment by Bluscream** [comment]
Not talking about the plugin api here, more about hooking qt
